tools/acncli/Dockerfile

```diff
@@ -1,22 +1,22 @@
-FROM golang as build
+FROM golang:1.17 as build
 WORKDIR /go/src/github.com/Azure/azure-container-networking/
-ADD . . 
 ARG VERSION
+ARG PLATFORM
+ADD . . 
 RUN make all-binaries
 RUN make acncli
 RUN rm -rf ./output/windows*
-RUN rm -rf ./output/linux_amd64/npm/*
+RUN rm -rf ./output/${PLATFORM:-linux_amd64}/npm/*
```

Comment:
I know Matt Long told me once it's best to copy in the source code and build it as part of the Dockefile.
Can we do that here too like in CNS?

Reply:
generally, that's true, because it makes the build consistent everywhere (because it's actually running in docker, everywhere). i don't think it's necessary here as that's not a component we are building/shipping to a variety of environments, but i would be willing to make the change